traveler_core_kit/src/androidTest/java/com/guestlogix/travelercorekit/TravelerTest.java

```diff
@@ -31,6 +33,8 @@
 @RunWith(AndroidJUnit4.class)
 public class TravelerTest{
     private static final String TAG = "TravelerTest";
+    private static final String TRAVELLER_ID = "eefda674-bb01-4c6c-b0fa-46f30341bf4f";
+    private static final String KENSINGTON_2H_TOUR_PRODUCT_ID = "ite_hkVh9VhhvV11nk_";
```

Comment:
I like this approach personally.

But I think these may change and this puts our tests at the mercy of the supplier